src/libsync/clientsideencryption.cpp

```diff
@@ -1260,6 +1335,38 @@ void ClientSideEncryption::getPublicKeyFromServer(const AccountPtr &account)
     job->start();
 }
 
+void ClientSideEncryption::getServerPublicKeyFromServer(const AccountPtr &account)
+{
+    qCInfo(lcCse()) << "Retrieving public key from server";
+    auto job = new JsonApiJob(account, baseUrl() + "server-key", this);
+    connect(job, &JsonApiJob::jsonReceived, [this, account](const QJsonDocument& doc, int retCode) {
+            if (retCode == 200) {
+                auto serverPublicKey = doc.object()["ocs"].toObject()["data"].toObject()["public-key"].toString();
+                qCInfo(lcCse()) << serverPublicKey;
```

Comment:
@mgallien this will look weird when reading logs later. I propose you move this to the next log line so we can then read something like **"serverPublicKey: ab2e51b0ce3da4..."**